pyomo/common/timing.py

```diff
@@ -198,22 +218,33 @@ def tic(self, msg=_NotSpecified,
             msg (str): The message to print out.  If not specified, then
                 prints out "Resetting the tic/toc delta timer"; if msg
                 is None, then no message is printed.
+            *args (tuple): optional positional arguments used for
+                %-formatting the `msg`
             ostream (FILE): an optional output stream (overrides the ostream
                 provided when the class was constructed).
             logger (Logger): an optional output stream using the python
                 logging package (overrides the ostream provided when the
                 class was constructed). Note: timing logged using logger.info
+            level (int): an optional logging output level.
 
         """
         self._lastTime = default_timer()
         if msg is _NotSpecified:
             msg = "Resetting the tic/toc delta timer"
         if msg is not None:
-            self.toc(msg=msg, delta=False, ostream=ostream, logger=logger)
-
-
-    def toc(self, msg=_NotSpecified, delta=True,
-            ostream=_NotSpecified, logger=_NotSpecified):
+            if args and '%' not in msg:
+                deprecation_warning(
+                    "'ostream', and 'logger' should be specified "
```

Comment:
Small peeve... I don't like the comma between `'ostream', and 'logger'`. Don't need to fix but it annoys me.